api/v1alpha1/managednotification_types.go

```diff
@@ -48,6 +51,9 @@ type Notification struct {
 	// The body text of the Service Log notification when the alert is resolved
 	ResolvedDesc string `json:"resolvedBody,omitempty"`
 
+	// Resources useful for context or remediation
+	Resources []NotificationResourceType `json:"resources,omitempty"`
```

Comment:
Can you expand the comment a bit more on what this should be? I see from lines 37-39 this should be a URL, but maybe something like this would be a bit more descriptive at first glance:

```suggestion
	// Resources useful for context or remediation - this could be links to documentation, KB articles, etc
	Resources []NotificationResourceType `json:"resources,omitempty"`
```